client-src/elements/chromedash-guide-metadata-page.js

```diff
@@ -0,0 +1,213 @@
+import {LitElement, css, html} from 'lit';
```

Comment:
This file is very close to `chromedash-guide-stage-page.js`, but does not query for stage data and attempt to use Stage entity information. I had considered trying to make that page work with both metadata and stage information, but it seemed that the logic would be unnecessarily complex when trying to accommodate both use cases, and this felt more easy to come back and refactor/edit.

Reply:
That's fine, we can look for ways to refactor commonalities later.